StRoot/StSpinPool/StJetEvent/StJetEvent.h

```diff
@@ -82,7 +82,7 @@ class StJetEvent : public TObject {
   TClonesArray* mTowers;
   TClonesArray* mParticles;
 
-  ClassDef(StJetEvent,6);
+  ClassDef(StJetEvent,7); //incremented to 7 for new updates with TOF information
```

Comment:
This change is not needed.
```suggestion
  ClassDef(StJetEvent,6);
```

Reply:
StJetEvent structure isn't changed, but StJetTrack, I kept Bassam change originally just to hope this will add an additional alert right away if you used an older version of StJetEvent code, the jet tree may not be compatible. But I don't have strong objection to this.

Reply:
@zlchang I see your point. The version numbers don't hurt or help anything, so we might use them to indicate major changes. I would not keep the comment though.